client/app/js/snackbar.jsx

```diff
@@ -0,0 +1,77 @@
+/*jshint esversion: 6 */
+import Snackbar from 'material-ui/Snackbar';
+import React from 'react';
+
+class MySnackbar extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: false,
+      message: "",
+      ws: undefined,
+      boardName: this.props.boardName,
+    };
+  }
+
+  handleTouchTap = () => {
+    this.setState({
+      open: true,
+      message: "Board Updated",
+    });
+  };
+
+  handleRequestClose = () => {
+    this.setState({
+      open: false,
+    });
+    this.boardReloadHandler();
+  };
+
+  boardReloadHandler = () => {
+    this.props.boardReloadHandler();
+  };
+
+  update_websocket = (board_name) => {
+    if (board_name === "LOADING...") {
+      return;
+    }
+    if (board_name.charAt(0) != "/") {
+      board_name = "/" + board_name;
+    }
+    var port = (location.port ? ':' + location.port : '')
+    var uri = window.location.hostname + port;
+    var socket = new WebSocket("ws://" + uri + board_name + "/ws");
+    socket.onmessage = this.handleTouchTap;
```

Comment:
I think the handler should have a different name. This has nothing to do with touch or clicking :)